tests/assemble.py

```diff
@@ -121,6 +121,20 @@ def test_assemble_global():
     assert exported_symbols == [(0, 'counter'), (1, 'internal'), (2, 'entry')]
 
 
+def test_assemble_uppercase_opcode():
+    a = Assembler()
+    try:
+        a.assemble("  WAIT 42")
+    except Exception as e:
```

Comment:
can we use a more specific exception here?

Reply:
I kept what was raised before, but I guess a `ValueError` exception could be nice here. I will change it.

Reply:
ok, great.